docs/design-document/03-connect/00-connectors.md

```diff
@@ -2,22 +2,49 @@
 
 ## Connector
 
-A connector is a bridge that interacts with a specific external service or underlying data source (e.g., Databases) on behalf of user applications. A connector is either a Source or a Sink.
+A connector is an image or instance that interacts with a specific external service or underlying data source (e.g., Databases) on behalf of user applications. A connector is either a Source or a Sink.
 
 ## Source
 
 A source connector obtains data from an underlying data producer, and delivers it to targets after original data has been transformed into CloudEvents. It doesn't limit the way how a source retrieves data. (e.g., A source may pull data from a message queue or act as an HTTP server waiting for data sent to it).
 
-CloudEvents is a specification for describing event data in common formats to provide interoperability across services, platforms and systems.
-
 ## Sink
 
 A sink connector receives CloudEvents and does some specific business logics. (e.g., A MySQL Sink extracts useful data from CloudEvents and writes them to a MySQL database).
 
+## CloudEvents
+
+A specification for describing event data in common formats to provide interoperability across services, platforms and systems.
+
 ## Implements
 
 Add a new connector by implementing the source/sink interface using [eventmesh-openconnect-java](https://github.com/apache/eventmesh/tree/master/eventmesh-openconnect/eventmesh-openconnect-java).
 
+## Technical Solution
+
+### Structure and process
+![source-sink connector architecture](https://github.com/apache/eventmesh/assets/13237619/e1897cd6-cc91-4dc4-b6d8-facd7b0538b3)
+
+### Design Detail
+![eventmesh-connect-detail](https://github.com/apache/eventmesh/assets/13237619/bc90925d-8503-4f32-8b5c-5ebc10c13c62)
+
+### Describe
+**Worker**
+
```

Comment:
`Worker` can be styled as a level-four heading.

Reply:
fixed